sdk/include/reactos/stubs.h

```diff
@@ -9,24 +31,30 @@
 #define EXCEPTION_WINE_STUB     0x80000100
 #define EH_NONCONTINUABLE       0x01
 
-NTSYSAPI
-VOID
-NTAPI
+/* __int128 is not supported on x86, so use a custom type */
+typedef struct
+{
+    __int64 lower;
+    __int64 upper;
+} MyInt128;
+
+void
+__stdcall
 RtlRaiseException(
-    _In_ PEXCEPTION_RECORD ExceptionRecord
+    PEXCEPTION_RECORD ExceptionRecord
 );
 
 ULONG
 __cdecl
 DbgPrint(
-    _In_z_ _Printf_format_string_ PCSTR Format,
+    const char* Format,
     ...
 );
 
 #define __wine_spec_unimplemented_stub(module, function) \
 { \
     EXCEPTION_RECORD ExceptionRecord = {0}; \
-    ExceptionRecord.ExceptionRecord = NULL; \
+    ExceptionRecord.ExceptionRecord = 0; \
```

Comment:
Which definition of `EXCEPTION_RECORD` structure it uses there? The one with `struct _EXCEPTION_RECORD *ExceptionRecord` member ? If so, why not using `((void *)0)` here?

Reply:
Because 0 is C++ compatible.